Class1/SortColor.java

```diff
@@ -0,0 +1,40 @@
+package Class1;
+
+// https://leetcode.com/problems/sort-colors/
+// Start 10:48
+// End and pass LC test case: 10:55
+public class SortColor {
```

Comment:
can you think of an alternative (completely different) solution and post that for me once you have it?

Reply:
Alternative solution:
1. Arrays.sort Time O(Nlog(N)) Space O(1)
2.  Create a Treemap(Red-black tree): key is the number and value is the frequency, then iterate each entry and write then to array. Treemap insert/delete are both O(log(n) total time is O(Nlog(N)) Space is O(N)